src/pages/Contents/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+import { SidebarItems } from "./SidebarItems";
+
+export const Sidebar = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleSidebar = () => {
+    setIsOpen(!isOpen);
```

Comment:
```suggestion
    setIsOpen(prev=>!prev);
```
이전의 상태을 기반으로 업데이트하는 함수형 업데이트를 찾아보세요~

Reply:
🪄 넵